### DISCUSSION TITLE
media upload error!!!

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#1068] [Q&A]
In order to get the original width and height of the upload, I follow the method provided by the documentation：

<img width="1476" alt="image" src="https://user-images.githubusercontent.com/16400882/188560229-41471e26-ca56-4024-8c84-fb4aa091cbbb.png">

![image](https://user-images.githubusercontent.com/16400882/188560470-c79a8bc8-f6e6-4ae7-add6-051fdbf82a6d.png)

But uploading the image throws an error：
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/16400882/188560895-f1f1f2b4-be9c-4f28-96e5-4847d130e528.png">

<img width="1219" alt="image" src="https://user-images.githubusercontent.com/16400882/188560934-bfa9d6cd-d84b-4706-9d95-69e1d7265c30.png">

**Comment by blueMountain007** [comment]
https://github.com/lovell/sharp/issues/1578

**Comment by jmikrut** [comment]
Hey @zhounewz — question for you. Do you get this message ONLY when you define width and height as null on your `original` image size? Can you upload the image without the width and height set?
